packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -168,22 +173,17 @@ abstract class MaterialLocalizations {
   /// each supported layout.
   TimeOfDayFormat timeOfDayFormat({ bool alwaysUse24HourFormat = false });
 
-  /// Provides geometric text preferences for the current locale.
-  ///
-  /// This text theme is incomplete. For example, it lacks text color
-  /// information. This theme must be merged with another text theme that
-  /// provides the missing values.
+  /// Defines the localized [TextStyle] geometry for [ThemeData.textTheme].
   ///
-  /// Typically a complete theme is obtained via [Theme.of], which can be
-  /// localized using the [Localizations] widget.
+  /// The [scriptCategory] defines the overall geometry of a [TextTheme] for
+  /// the static [MaterialTextGeometry.localizedFor] method in terms of the
+  /// three language categories defined in https://material.io/go/design-typography.
```

Comment:
nit: urls can contain dots, and mean something different with them at the end than without.
a few workarounds:
* `three language categories defined in: https//example.com`
* `three language categories defined in <https//example.com>.`
* `[three language categories](https//example.com).`

Reply:
Good point. I've fixed up the naked URLs.